src/pages/guidelines/content/writing-style.mdx

```diff
@@ -3,15 +3,15 @@ title: Content
 description:
   Writing should always be simple, clear, and easy to understand. Using everyday
   language keeps the tone friendly, human, and inviting. And choose short words
-  over long impressive-sounding words. Put the thesaurus away!
+  over long impressive–sounding words. Put the thesaurus away!
 tabs: ['Overview', 'Writing style', 'Action labels']
 ---
 
 <PageDescription>
 
 Writing should always be simple, clear, and easy to understand. Using everyday
 language keeps the tone friendly, human, and inviting. And choose short words
-over long impressive-sounding words. Put the thesaurus away!
+over long impressive–sounding words. Put the thesaurus away!
```

Comment:
Unrelated to hyphens, but there needs to be a comma between the two coordinate adjectives: long, impressive-sounding...